src/Admin.php

```diff
@@ -28,9 +28,38 @@ public static function init() {
     // Adds products variations custom fields.
     add_action('woocommerce_product_after_variable_attributes', __NAMESPACE__ . '\WooCommerce::woocommerce_product_after_variable_attributes', 10, 3);
 
+    // Allow ajax requests to specified functions.
+    add_action('wp_ajax_is_existing_gtin', __NAMESPACE__ . '\WooCommerce::ajax_is_existing_gtin');
+
+    // Enqueues admin plugin scripts.
+    add_action('admin_enqueue_scripts', __CLASS__ . '::admin_enqueue_scripts');
+
     Settings::init();
   }
 
+  /**
+   * Enqueues admin plugin scripts.
+   *
+   * @implements admin_enqueue_scripts
+   */
+  public static function admin_enqueue_scripts($hook) {
+    $git_version = Plugin::getGitVersion();
+    global $post;
+    if ($hook == 'post-new.php' || $hook == 'post.php') {
+      if ($post->post_type === 'product') {
+        wp_enqueue_script(Plugin::PREFIX . '_admin', Plugin::getBaseUrl() . '/dist/scripts/admin.min.js', ['jquery'], $git_version, TRUE);
+        wp_localize_script(Plugin::PREFIX . '_admin', 'shop_standards_admin', array(
+          'product_id' => $post->ID,
+          'gtin_error_message' => sprintf(
+            __('The entered GTIN <a href="%s">exists already</a>. It must be changed in order to save the product.', Plugin::L10N),
```

Comment:
I do not understand why the localized error message is different from the message shown on `WooCommerce.php:235` and `WooCommerce.php:413`. Can we not extract all the message errors into a single method and apply the same translation to all of them? 🤔

Reply:
The context is a little different: this message is used for AJAX validation (before anything is saved), the other one is shown after all other data has been saved – it's only a slight difference, but I still think it makes sense to differentiate. 
I might add the edit link of the product where the duplicate has been found to the notice after server side validation as well though, to add some value. What do you think?

Reply:
@moritzbappert — still does not agree about the difference between AJAX and non-AJAX messages. But I will leave this choice up to you.

Anyway, yes: agree about the edit link of the product, if possible.

Also, please adjust the message:

```diff
- __('The entered GTIN <a href="%s">exists already</a>. It must be changed in order to save the product.', Plugin::L10N),
+ __('The entered GTIN <a href="%s">already exists</a>. It must be changed in order to save the product.', Plugin::L10N),
```